packages/selleo-tailwind/tailwind.config.cjs

```diff
@@ -14,10 +14,41 @@ module.exports = {
       8: "144px",
       9: "192px",
     },
+    colors: {
+      white: '#ffffff',
+      black: '#2F3033',
+      neutral: {
+        100: '#ff6d2a',
+        200: '#eaeaeb',
+        300: '#bfc0c4',
+        400: '#95979d',
+        500: '#696b72',
+        600: '#4c4d52',
+      },
+    },
+    letterSpacing: {
+      normal: '0.01em',
+    },
     extend: {
       colors: {
-				'primary-500': '#FAFAFA',
-			}
+        brand: {
+          primary: {
+            DEFAULT: '#ff6d2a',
```

Comment:
i think i'd get rid of DEFAULT key in the colors
 we can leave a comment next to 500 key `// default`

Reply:
Done